### PR TITLE
feat: comparison history

### PR DESCRIPTION
**Pull request by kobryl**

### CODE REVIEW

**Review by michalkr52 — APPROVED** [review]
LGTM, but I'd hold off with merging until I prepare the frontend for this change (and also test it)

File: objects_counter/migrations/versions/95936b76d801_add_comparison_history.py
Comment (kobryl) [inline objects_counter/migrations/versions/95936b76d801_add_comparison_history.py:13]: Check with #141 before merging

**Pull request merged**
